packages/cli/test-utils/fixtures.ts

```diff
@@ -53,6 +54,25 @@ export class TestFixture {
       transformResults
     } = this.environment;
 
+    // Default to local templates to avoid network and keep tests fast
+    const hasInitCommand = Array.isArray(argv._) && argv._.includes('init');
+    if (hasInitCommand) {
+      argv.repo = argv.repo ?? DEFAULT_REPO;
+      argv.templatePath = argv.templatePath ?? (argv.workspace ? 'workspace' : 'module');
+      const defaults = {
+        fullName: 'Tester',
+        email: 'tester@example.com',
+        moduleName: argv.workspace ? 'starter-module' : argv.name || argv.MODULENAME || 'module',
```

Comment:
this code looks copy and pasted.... let's deduplicate all 3 (or more) of the places this code is used, please.